source/common/http/conn_manager_config.h

```diff
@@ -213,6 +213,11 @@ class ConnectionManagerConfig {
    */
   virtual bool generateRequestId() PURE;
 
+  /**
+   * @return whether the x-request-id should be reset on edge entry inside mesh
```

Comment:
Nit: whether the x-request-id should not be reset..

Reply:
done